evennia/contrib/rpg/rpsystem/rpsystem.py

```diff
@@ -1679,8 +1533,36 @@ def at_pre_say(self, message, **kwargs):
 
         """
         if kwargs.get("whisper"):
-            return f'/me whispers "{message}"'
-        return f'/me says, "{message}"'
+            return f'/Me whispers "{message}"'
+        return f'/Me says, "{message}"'
+
+    def get_sdesc(self, obj, process=False, **kwargs):
+        """
+        Single hook method to handle getting recogs with sdesc fallback in an
+        aware manner, to allow separate processing of recogs from sdescs.
+        Gets the sdesc or recog for obj from the view of self.
+
+        Args:
+            obj (Object): the object whose sdesc or recog is being gotten
+        Keyword Args:
+            process (bool): If True, the sdesc/recog is run through the
+                appropriate process_X method.
```

Comment:
This is not very informative. What does it mean to run through the `process_X` methods? Why would you not want to use them? Are they just hooks to customize the output? Also, there are only two `process_X` methods: `.process_recog` and `.process_sdesc` - you can spell this out in full  (note the use of period to indicate they are on this class, but it can be good to also mention this explicitly).